Source/WebKit/UIProcess/Inspector/mac/WKInspectorViewController.mm

```diff
@@ -68,6 +68,7 @@ @implementation WKInspectorViewController {
     RetainPtr<WKInspectorWKWebView> _webView;
     WeakObjCPtr<id <WKInspectorViewControllerDelegate>> _delegate;
     RetainPtr<_WKInspectorConfiguration> _configuration;
+    BOOL _isAttached;
```

Comment:
Fairly certain this isn't necessary with auto-synthesis of properties. You'll get an _isAttached for free.